### PR TITLE
Where to find the key file

### PR DESCRIPTION
**Pull request by nahoj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,10 @@ In order to support the decryption, install pycryptodome if it is not installed
 pip install pycryptodome # Or 
 pip install whatsapp-chat-exporter["android_backup"] # install along with this software
 ```
+
 #### Crypt12 or Crypt14
+You will need the decryption key file. You can find it on your phone in the WhatsApp backup directory as `WhatsApp/Databases/.nomedia`, or, if you have root access, `/data/data/com.whatsapp/files/key`.
```
Comment (KnugiHK) [inline README.md:47]: The key will appear in WhatsApp/Databases/.nomedia now?

Comment (nahoj) [reply README.md:47]: For me it did yes.

Comment (KnugiHK) [reply README.md:47]: What's the name of the key file?

Comment (nahoj) [reply README.md:47]: `.nomedia` is the key file.

Comment (KnugiHK) [reply README.md:47]: Omg!!! I just realize it is the key file. That is a huge finding for me.

Comment (KnugiHK) [reply README.md:47]: Did you run the [WhatsApp-Key-DB-Extractor](https://github.com/KnugiHK/WhatsApp-Key-DB-Extractor/tree/master) before the `.nomedia` key file appear?

Comment (nahoj) [reply README.md:47]: I ran the extractor but it failed and I don't know if the `.nomedia` file was there before.

The extractor failed saying it didn't find a number of files under /data (IIRC). I closed the shell before I thought to save the output so I can't tell you more. When it restored the latest WhatsApp version it lost the icon and name (the app appeared as com.[...] in the menu) so I was a bit scared of losing my data and I didn't dare run it again.

Comment (KnugiHK) [reply README.md:47]: It is always good to be cautious when a situation like this happens. I will confirm whether `.nomedia` is there before the extractor does it work.

Comment (KnugiHK) [reply README.md:47]: I can confirm that the `.nomedia` key file is copied by [WhatsApp-Key-DB-Extractor](https://github.com/KnugiHK/WhatsApp-Key-DB-Extractor).

I suggest changing a bit of your description before I merge it. For example, "If you used WhatsApp-Key-DB-Extractor before....".

Comment (nahoj) [reply README.md]: Done. If you want to edit it, I don't mind at all if you discard my PR and just write it the way you think is best, it's your project. Thank you.

**Review by KnugiHK — COMMENTED** [review]
LGTM. (Although I left this review after I merged the PR😂)

**Pull request merged**